src/types/frequency.ts

```diff
@@ -0,0 +1,6 @@
+enum Frequency {
```

Comment:
I've been considering switching from enums to string unions (reduces runtime overhead), how do you feel about it?

```ts
type Frequency = "ALWAYS" | "ONCE";
```

Reply:
I have mixed feelings. I don't like how enums are transpiled in TS, so there is a clear advantage to string unions. I just don't like how the physically look, since they look like magic strings instead of constants. I think if I just don't use ALL CAPS, it'll look fine.

Reply:
@MeltingMosaic Yeah I agree, I wish enums in typescript were zero runtime overhead. Maybe we'll get lightweight enums eventually haha